docs/notebooks/neural_dual.ipynb

```diff
@@ -12,7 +12,7 @@
    "metadata": {},
```

Comment:
<p>The potentials are swapped, should be <code>\nabla f(source)</code>and <code>\nabla g(target)</code> to match the changes made in the code.</p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>